src/v/storage/tests/utils/log_gap_analysis.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#pragma once
+
+#include "model/fundamental.h"
+#include "model/record_batch_reader.h"
+
+namespace storage {
+
+// Utility to analyze offset gaps in logs, for testing compaction logic.
+
+struct log_gap_analysis {
+    model::offset first_gap_offset{-1};
```

Comment:
Can you please add a comment when `first_gap_offset` and `last_gap_offset` mean?

Reply:
Added a comment explaining the fields, thanks. Update coming soon.